### PR TITLE
*: register cdc/resolved_ts endpoint in server2

### PR DESCRIPTION
**Pull request by overvenus**
### What is changed and how it works?

Issue Number: ref #14542

What's Changed:

* Register cdc/resolved_ts endpoint in server2
* Add `CdcHandle` trait to make cdc and resolved_ts compatible with raftstore v2.

```commit-message
Register cdc/resolved_ts endpoint in server2
```

### Check List 

Tests 

- Unit test
- Integration test

### Release note 

```release-note
None
```

### CODE REVIEW

File: components/raftstore/src/router.rs [hunk 1]
```diff
@@ -384,3 +384,68 @@ impl<EK: KvEngine, ER: RaftEngine> crate::coprocessor::StoreHandle for RaftRoute
         );
     }
 }
+
+/// A handle for cdc and pitr to schedule some command back to raftstore.
+pub trait CdcHandle<EK>: Clone + Send
```
Comment (hi-rustin) [inline components/raftstore/src/router.rs:389]: Maybe we need to find a better place for this trait. Because raftstore-v2 also implements it.

**Review by hi-rustin — APPROVED** [review]
Looks good to me. Thanks!

**Review by 3pointer — APPROVED** [review]

File: components/raftstore/src/router.rs (hunk 1 above)
Comment (overvenus) [reply components/raftstore/src/router.rs:389]: raftstore-v2 imports raftstore, I think it's fine.

**Review by Connor1996 — APPROVED** [review]

**Pull request merged**